common-data/norent-state-law-for-builder.json

```diff
@@ -0,0 +1,5 @@
+{
+  "SD": {
+    "Text of Legislation": "Tenants in Florida are protected from eviction for non-payment by Executive Order 20-94, issued by Governor Ron DeSantis on April 2, 2020."
+  }
```

Comment:
For some reason I guess the legislation for Florida was set to South Dakota in the Airtable at the time I made this--it is _not_ a bug in the code of this PR.